aws-iam-policies/reduced-permissions-mode.json

```diff
@@ -15,7 +15,10 @@
                 "autoscaling:UpdateAutoScalingGroup",
                 "cloudformation:DescribeStackEvents",
                 "cloudformation:UpdateStack",
-                "ec2:DeleteKeyPair"
+                "ec2:DeleteKeyPair",
+                "rds:StartDBInstance",
+                "rds:StopDBInstance",
```

Comment:
Don't think those action support this condition. Pls follow same as restricted policy.